### DISCUSSION TITLE
Comparison to microcks

### DISCUSSION THREAD

**Discussion by hideintheclouds** [specmatic/specmatic#1990] [General]
Would be interesting to know how does Specmatic compare to [microcks](https://microcks.io/documentation/using/openapi/)
https://medium.com/@lbroudoux/microcks-and-pact-for-api-contract-testing-3e0e7d4516ca

They seem to offer similar capabilities.
A similar comparison like the one to [ pact](https://specmatic.in/comparisons/specmatic-vs-pact-io-and-pactflow-io/#gsc.tab=0) would be appreciated

**Comment by yogeshnikam671** [comment]
Hi @hideintheclouds, thanks for suggesting this idea.
We will get back on this.

**Comment by harikrishnan83** [comment]
Hello @hideintheclouds , here is the comparison - https://specmatic.io/comparisons/specmatic-vs-microcks/. Thank you for your patience on this. Regards.
